docs/controls/datagrid_guidance/styling_formatting_options.md

```diff
@@ -54,7 +54,23 @@ All individual parts of the DataGrid control are customizable through simple Sty
    * *RowStyle* : style that is used when rendering the rows.
    * *ColumnHeaderStyle* : style that is used when rendering the column headers.
    * *RowGroupHeaderStyles* : style that is used when rendering the row group header. The DataGrid control currently supports one-level grouping only - the first style specified as part of RowGroupHeaderStyles will be applied to the row group header.
-   * *RowHeaderStyle* : style that is used when rendering the row headers. The DataGrid control does not have a default visual for the row header. If you would like to alter a row header visual, you should first provide one by altering the DataGrid's control template.
+   * *RowHeaderStyle* : style that is used when rendering the row headers. The DataGrid control does not have a default visual for the row header. To provide one you must provide a `ContentTemplate` for `Microsoft.Toolkit.Uwp.UI.Controls.Primitives.DataGridRowHeader`. For Example: 

```

Comment:
Content template or control template? Although I suppose both are accurate, just wasn't sure if you wanted to be more specific about it.

Reply:
"control template" seems generic and so I wanted to specifically call out the type of template that must be created as indicated in the example below.

Reply:
Sound good, fine with leaving it like that.